resources/europe/switzerland/osm-ch.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "osm-ch",
-  "type": "osm-lc",
+  "type": "url",
```

Comment:
So, `url` is accurate, but `osm-lc` is also one of the [valid types](https://github.com/osmlab/osm-community-index/wiki/Contributing#type-required-account-optional). It’s for official OSMF local chapter websites. Which would be preferable?

Reply:
The website is operated by the LV, but isn't the LC site which has its own independent entry.